packages/next-app/src/app/stableswapsimulator/(components)/ImpactCurve.tsx

```diff
@@ -72,9 +72,17 @@ export function ImpactCurve() {
       decimals,
     });
 
+    const amountsOut = amountsIn.map(
+      (amount) =>
+        MetaStableMath.exactTokenInForTokenOut(
+          MetaStableMath.numberToOldBigNumber(amount),
+          poolPairDataOut
+        ).toNumber() * -1
+    );
+
     const impactTabTokenIn = amountsIn.map(
       (amount) =>
-        MetaStableMath.priceImpactForExactTokenOutReversedSwap(
+        MetaStableMath.priceImpactForExactTokenInReversedSwap(
```

Comment:
in this case, we have the "out" that is the `amountsIn` and we want to get the price Impact from that, so isn't the correct function?

![image](https://github.com/bleu-studio/balancer-pool-metadata/assets/63298341/2ead2b28-c68c-4904-a527-0b09a9bd15d0)

Reply:
Note:
- Improve code readability using 2 times the calculated amounts (one for token in and another one for out). Improve names as well.